watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py

```diff
@@ -1310,26 +1309,30 @@ def model6():
 @pytest.mark.unit
 def test_elec_properties_errormsg(model6):
     m = model6
-    with pytest.raises(
-        ConfigurationError,
-        match="The charge property should not be assigned to the neutral component",
-    ):
-        m[0].fs.properties = MCASParameterBlock(
-            solute_list=["Na_+", "Cl_-", "N"],
-            mw_data={"H2O": 0.018, "Na_+": 0.023, "Cl_-": 0.0355, "N": 0.01},
-            charge={"N": 0, "Na_+": 1, "Cl_-": -1},
-        )
+    # with pytest.raises(
+    #     ConfigurationError,
+    #     match="The charge property should not be assigned to the neutral component",
+    # ):
+    #     m[0].fs.properties = MCASParameterBlock(
+    #         solute_list=["Na_+", "Cl_-", "N"],
+    #         mw_data={"H2O": 0.018, "Na_+": 0.023, "Cl_-": 0.0355, "N": 0.01},
+    #         charge={"N": 0, "Na_+": 1, "Cl_-": -1},
+    #     )
     m[0].fs.properties = MCASParameterBlock(
         solute_list=["Na_+", "Cl_-", "N"],
         mw_data={"H2O": 0.018, "Na_+": 0.023, "Cl_-": 0.0355, "N": 0.01},
         charge={"Na_+": 1, "Cl_-": -1},
         elec_mobility_calculation=ElectricalMobilityCalculation.EinsteinRelation,
+        diffus_calculation=DiffusivityCalculation.HaydukLaudie,
```

Comment:
I don't see why this is added, only if you are writing a new test  but what is about this original test?

Reply:
Well, first let me explain why I included a default value for diffus_phase_comp when DiffusivityCalculation is None. Before, as a parameter, we could get away with the user not providing a value, but when I add_object_reference within the diffus_phase_comp method for the stateblock, an error will occur. Thus, I decided for calculation type of None, it wouldn't hurt to assign a default and provide a logger warning. I get why we think that might be risky, but I'd prefer this way rather than having separately named params/vars to point to when choosing between diffus_phase_comp on stateblock or parameter block.

However, I preserved the functionality behind Hayduk-Laudie. So with no diffus calculation, this error message below will not be raised because default values are provided in absence of a user-provided value (again, because it is not a fixed var on parameter block that is also referenced in the associated on-demand method). 

By specifying Hayduk-Laudie, the error message _will_ be raised, since no defaults need be supplied.

Reply:
I don't think that is how I wrote the code- error message was controlled by `diffusivity_data` in the configuration rather than any parameter or vars.  The error should show in any methods to assure the correctness of the electrical property calculation.

Reply:
> I get why we think that might be risky, but I'd prefer this way rather than having separately named params/vars to point to when choosing between diffus_phase_comp on stateblock or parameter block.

1) I don't think naming is more important than avoiding a risk here; 2) the name could be changed to the same in the previous version; 3) we decided to eliminate the risk and was done in the previous version; 4) diffusivity is critical, e.g., is related to electrical migration properties- it had occurred that the default values messed up some calculations in ED models that were later caught by me. I imagine this can easily happen to users so I still recommend eliminating it.